src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -16,8 +16,8 @@
     position: fixed;
     top: rem(90px);
     height: 100%;
-    background-color: #fff;
-    border-right: 1px solid rgba(#c2c0c0, 0.5);
+    background-color: $ui-02;
+    border-right: 1px solid rgba($active-ui, 0.5);
```

Comment:
- [ ] I think this might cause some issues with the CSS variables. These might have to be two hard coded hex values that get overridden using the `bx--v9-theme` as the use of a function with the CSS custom properties flag in carbon enabled can cause issues as it expects a hex and receives a variable instead 😅. Open to other solutions though 👍

Reply:
I've made a change, but I'm not sure with what values should I overwrite the colors when v9 theme is enabled

Reply:
@Bgazsi When v9 is enabled it should just be the original colors. When v10 is enabled I would just make sure it looks consistent. If you need help with color matching you can post images in our team slack channel and they should be able to respond pretty quick. Most teams should be moving off of this component in the future in favor of adopting the L2 and L3 navs.

Reply:
Thanks, I've posted a screenshot, please let mi know if there's anything else I can do 🙂